Automattic/jetpack
# VideoPress: fix error when uploading tracks in Atomic sites

Fixes #

## Proposed changes:

* VideoPress: fix error when uploading tracks in Atomic sites

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test in an Atomic site
* Go to the block editor
* Create/Edit a VideoPress video block
* Upload a track
* Confirm it doesn't fail. tracks request doesn't return an error. The track is uploaded correctly.

<img width="855" alt="Screen Shot 2023-01-26 at 16 04 04" src="https://user-images.githubusercontent.com/77539/214886160-e4577e9c-a0b7-4b8d-8c37-d858fe14afaf.png">
<img width="854" alt="Screen Shot 2023-01-26 at 16 03 44" src="https://user-images.githubusercontent.com/77539/214886168-488ccee7-75b3-42a1-8e26-1044d971df55.png">
